hera_mc/observations.py

```diff
@@ -76,3 +76,8 @@ def create(cls, starttime, stoptime, obsid, hera_cofa):
         return cls(obsid=obsid, starttime=starttime.gps, stoptime=stoptime.gps,
                    jd_start=starttime.jd,
                    lst_start_hr=starttime.sidereal_time('apparent').hour)
+
```

Comment:
Do we have to create a new view table object for every table?  I was hoping we could handle this transparently so that new tables will show up without having to manually add the matching view, or without having to go through and manually add views for each table.

Reply:
Yes, we have to make one for each table. But I put all the hard work in the `__init__.py`, so it's a 3-liner to add it for any new table and I can add it to the list of steps for making a new table in the readme. There may be a more clever way to do it, but I don't know how.

Reply:
Yes it is. I made one and here's the output, noted as:
table name: time columns

Note that several of the listed columns are not in gps time and it's not super straightforward to infer from code which ones are and which ones aren't in gps seconds.
```
Tables with missing views:

lib_server_status: mc_time, mc_system_timediff, uptime_days
lib_status: time
autocorrelations: time
dubitable: start_gpstime, stop_gpstime
part_info: posting_gpstime
cm_version: update_time
subsystem_error: time, mc_time
parts: start_gpstime, stop_gpstime
lib_raid_status: time
lib_raid_errors: time
lib_remote_status: time, ping_time
rtp_server_status: mc_time, mc_system_timediff, uptime_days
rtp_status: time
weather_data: time
node_sensor: time
node_power_status: time
node_power_command: time
correlator_control_state: time
correlator_control_command: time
connections: start_gpstime, stop_gpstime
geo_location: created_gpstime
lib_files: time
rtp_process_event: time
rtp_process_record: time
rtp_task_resource_record: start_time, stop_time
ant_metrics: mc_time
array_metrics: mc_time
correlator_take_data_arguments: time, starttime_sec, starttime_ms, integration_time
correlator_config_status: time
correlator_config_command: time
```

Reply:
I can make this test part of our standard tests if you like. Because of some of the vagaries of the column naming, I don't see a good way to use this to make views by default, but I can keep thinking about that, maybe I'll come up with something.